TACC/protx-dashboard
# task/WG-17: update to handle demographic features from cooks 371

## Overview: ##

This PR:
* bumps the cooks.db to latest version (cooks_20230505.db)
* and adds a note below demographic chart for any demographic features (currently 3 demographic values note for 3 demographic

## Related Jira tickets: ##

* [WG-17](https://jira.tacc.utexas.edu/browse/WG-17)

## Testing Steps: ##
0. Get cooks_20230505.db
1. Go to Demographics and select a county
2. Review "Two or more races", "Single parent households" and "Minority population
3. Review the extra note placed below the chart
4. Go to Analytics to ensure the view community characteristics modal works.
